### ISSUE TITLE
Pending RDS Required OS Update

### ISSUE THREAD

**Issue by jkueloc** [LibraryOfCongress/concordia#1571]
Description:  New Operating System update is available
Type:  system-update
Status:  required	
Apply date: January 10, 2022, 12:00:00 AM UTC

This ticket is to capture that initial work that has gone into researching the unannounced (no email) pending RDS OS maintenance and applying the OS upgrade to at least the development environment ahead of the required maintenance date.

Research - When I saw the notice on the AWS RDS console, I checked with Rosie to see if she had received any emails  - she checked and found none. I googled about it and also searched AWS and couldn't find anything specific to what to do to prepare or expect with this required OS update that the RDS (database) sits - something that AWS email notices would typically cover in an email with links to the related documents.

I realize that there are no prod equivalent environments - so there is only so much that would reveal itself as a possible issue. 

That said see if there are noticeable impacts in dev. We'll likely need to open support ticket with AWS since as I understand it, these types of updates should be handled by the platform provider (AWS).

Will schedule the Development update for this weekends maintenance window..... if it will let me...

**Comment by acdha** [comment]
Emails appear to be in the process of rolling out now to the account owner and technical contacts. If you want to have other notifications, here are instructions for configuring those using SNS:

https://aws.amazon.com/premiumsupport/knowledge-center/notification-maintenance-rds-redshift/

**Comment by jkueloc** [comment]
Update - @acdha I did set up event notification for maintenance, notification, etc. to my email.  Thank you! 

Seems like the console may have been just being sluggish or non-communicative earlier when I applied it to the next maintenance window and the console didn't register anything.  
Now the console shows the pending required OS update as scheduled for the "next window" on November 21, 2021, 7:14:00 AM UTC. 

I did not use the CLI to apply the pending maintenance to the next maintenance window for development but was preparing to take that walk...
